otx/algorithms/action/adapters/mmaction/models/heads/movinet_head.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import torch.nn as nn
+from mmcv.cnn import normal_init
+from mmaction.models.builder import HEADS
+from mmaction.models.heads.base import BaseHead
+from ..backbones.movinet import ConvBlock3D, Swish
+
+
+@HEADS.register_module()
+class MoViNetHead(BaseHead):
+    """Classification head for MoViNet.
+    Args:
+        num_classes (int): Number of classes to be classified.
+        in_channels (int): Number of channels in input feature.
+        loss_cls (dict): Config for building loss.
+            Default: dict(type='CrossEntropyLoss')
+        spatial_type (str): Pooling type in spatial dimension. Default: 'avg'.
+        dropout_ratio (float): Probability of dropout layer. Default: 0.5.
+        init_std (float): Std value for Initiation. Default: 0.01.
+        fc1_bias (bool): If the first fc layer has bias. Default: False.
+    """
+
+    def __init__(self,
+                 num_classes: int,
+                 in_channels: int,
+                 hidden_dim: int,
+                 tf_like: bool = False,
+                 conv_type: str = '3d',
+                 loss_cls=dict(type='CrossEntropyLoss'),
+                 spatial_type: str = 'avg',
+                 dropout_ratio: float = 0.5,
+                 init_std: float = 0.1,
+                 causal: bool = False):
+        super().__init__(num_classes, in_channels, loss_cls)
+
+        self.spatial_type = spatial_type
+        self.dropout_ratio = dropout_ratio
+        self.init_std = init_std
+
+        self.classifier = nn.Sequential(
+            # dense9
```

Comment:
This comment does not give useful information.
This can be removed

Reply:
Reflected, thank you for nice catch.